test/test_plugin_registry.py

```diff
@@ -33,53 +33,61 @@ class InvalidPlugin(GirderPlugin):
     pass
 
 
-class NoDeps(GirderPlugin):
+class TestLoadMixin:
     def __init__(self, *args, **kwargs):
-        super(NoDeps, self).__init__(*args, **kwargs)
+        super().__init__(*args, **kwargs)
```

Comment:
Can this line just be removed?

Reply:
No. In practice, this calls through to `GirderPlugin.__init__`, which is mandatory.